pages/1.11/release-notes/1.11.4/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.4
+title: Release Notes for 1.11.4
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.4
+---
+
+DC/OS 1.11.4 was released on July 26, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.4/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.4 includes the following:
+
+- Apache Mesos 1.5.2-dev [change log](https://github.com/mesosphere/mesos/blob/197c9105c2ed08f1529602f1ff72de846cde98c6/CHANGELOG).
```

Comment:
Let's point to https://github.com/apache/mesos/blob/2440c7310627c835fe828aef6b28b8d072e7371c/CHANGELOG

this sha matches what's in the `buildinfo.json` here https://github.com/dcos/dcos/blob/1.11.4/packages/mesos/buildinfo.json

Reply:
@vinodkone Done.